src/Plugin/TestsiteBuilder/ConfigImporter/EntityBrowser.php

```diff
@@ -1,187 +0,0 @@
-<?php
-
-namespace Drupal\testsite_builder\Plugin\TestsiteBuilder\ConfigImporter;
-
-use Drupal\config_update\ConfigRevertInterface;
-use Drupal\Core\Entity\EntityTypeManagerInterface;
-use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
-use Drupal\Core\Plugin\PluginBase;
-use Drupal\testsite_builder\ConfigImporterInterface;
-use Drupal\update_helper\ConfigName;
-use Symfony\Component\DependencyInjection\ContainerInterface;
-
-/**
- * Config importer plugin for entity_browser configuration type.
- *
- * @TestsiteBuilderConfigImporter(
- *   id = "entity_browser",
- *   label = @Translation("Generic"),
- *   description = @Translation("Generic config importer plugin.")
- * )
- */
-class EntityBrowser extends PluginBase implements ConfigImporterInterface, ContainerFactoryPluginInterface {
-
-  /**
-   * The entity manager service.
-   *
-   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
-   */
-  protected $entityTypeManager;
-
-  /**
-   * The config reverter service.
-   *
-   * @var \Drupal\config_update\ConfigRevertInterface
-   */
-  protected $configReverter;
-
-  /**
-   * List of entity browser widgets supported by configuration modifications.
-   *
-   * @var array
-   */
-  protected $supportedEntityBrowserWidgets = [
-    'dropzonejs_media_entity',
-    'dropzonejs_media_entity_inline_entity_form',
```

Comment:
We need this ConfigImporter at least for these dropzone widgets, to be able to upload images into the elsewhere defined media types (including their custom fields).
We lose this ability without this configuration and with the hacked in a media.type.image in src/ConfigCreator.php